client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
Can we use something more general here, like a `data-attr` we can use for popovers, tooltips, this, etc?